compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/res/DesktopSvgResources.desktop.kt

```diff
@@ -49,6 +49,7 @@ import org.jetbrains.skia.svg.SVGPreserveAspectRatioAlign
  * will be drawn with the specified size, density will have no effect.
  * @return the decoded SVG image associated with the resource
```

Comment:
> Chore - Resources

Let's change it to
```
Breaking changes - Resources
```

The values should be from this [list](https://github.com/JetBrains/compose-multiplatform-core/edit/jb-main/.github/PULL_REQUEST_TEMPLATE.md) (we extend it when needed)

Reply:
✅